.travis.yml

```diff
@@ -23,10 +27,21 @@ script:
                  --build-arg TRAVIS_PULL_REQUEST=${TRAVIS_PULL_REQUEST} --build-arg TRAVIS_JOB_ID=${TRAVIS_JOB_ID}
                  --build-arg TRAVIS_TAG=${TRAVIS_TAG} --build-arg TRAVIS_REPO_SLUG=${TRAVIS_REPO_SLUG}
                  --build-arg TRAVIS_COMMIT=${TRAVIS_COMMIT} --build-arg COVERAGE=${COVERAGE}
+                 --build-arg DOCS=${DOCS}
                  -t ${TRAVIS_REPO_SLUG}:latest ${HOME}/docker/
 
 after_success:
   - if [[ ${DOCKERHUB} = true && ${DOCKER_USERNAME} && ${DOCKER_PASSWORD} && ${TRAVIS_PULL_REQUEST} == false && ${TRAVIS_BRANCH} == master ]]; then
```

Comment:
You only compile with gcc, right? (If you ever enable `clang` you need to check it here.)

Reply:
That's correct, but good point.